package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "start": "node index.js",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
-    "lint": "prettier --check .",
-    "lint:fix": "prettier --write ."
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
```

Comment:
does this make it so you don't need to invoke prettier at all? i.e. the plugin will do prettier under the hood?

Reply:
Yup!

Reply:
however this only applies to js files 😅

So potentially you want it for other files

Reply:
https://github.com/timja/github-comment-ops/issues/57